### PR TITLE
Awa add stormpath

### PR DESCRIPTION
**Pull request by a-ashbeck**

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import ReactStormpath, { Router, HomeRoute, LoginRoute, AuthenticatedRoute } from 'react-stormpath';
```
Comment (cbrenner04) [inline app/app.js]: just out of curiosity, why are some in an object and others not?

File: app/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            <ul className="nav navbar-nav">
+              <li><Link to="/">Home</Link></li>
+              <Authenticated>
+                <li>
+                  <Link to="/profile">Profile</Link>
+                </li>
+              </Authenticated>
+            </ul>
+            <ul className="nav navbar-nav navbar-right">
+              <NotAuthenticated>
```
Comment (cbrenner04) [inline app/components/Header.js]: are these storm path components. Its wild seeing them used like this. Presumably they are just a link somewheres?

File: controllers/mainController.js [hunk 3]
```diff
@@ -1,5 +1,48 @@
+var bodyParser = require('body-parser');
```
Comment (cbrenner04) [inline controllers/mainController.js:1]: We should probably update this to ES6. I think being consistent will be very helpful down the line

File: server.js [hunk 4]
```diff
@@ -2,6 +2,9 @@ var bodyParser = require('body-parser');
 var express = require('express');
```
Comment (cbrenner04) [inline server.js:2]: again we should probably use ES6.

File: controllers/mainController.js [hunk 5]
```diff
@@ -1,5 +1,48 @@
...
+var stormpath = require('express-stormpath');
+
 module.exports = function(app) {
     app.get('/', function(request, response) {
         response.send('index.html');
     });
+
+    app.post('/me', bodyParser.json(), stormpath.loginRequired, function (req, res) {
+		  function writeError(message) {
```
Comment (cbrenner04) [inline controllers/mainController.js]: why is this indented so far?

**Review by cbrenner04 — COMMENTED** [review]
I'd like to figure out why some imports are in `{}` and others are not. Is it necessary or do we just need to find a consistent way. Did you run the linter?

We don't have to change to ES6 on this PR. I can work on it over the next couple of days.

File: app/app.js (hunk 1 above)
Comment (cbrenner04) [reply app/app.js]: just so everyone knows...  it definitely matters

File: app/config/routes.js [hunk 6]
```diff
@@ -0,0 +1 @@
+// routes?
```
Comment (cbrenner04) [inline app/config/routes.js]: @a-ashbeck is it possible we need some routes? I'm just grasping at straws. I will start looking into the API tomorrow

File: server.js [hunk 7]
```diff
@@ -17,14 +17,13 @@ app.use(bodyParser.json());
 // Parse application/x-www-form-urlencoded
 app.use(bodyParser.urlencoded({extended: false}));
 
-app.use(stormpath.init(app, {web: {produces: ['application/json']}}));
+app.use(stormpath.init(app, {website: true}));
```
Comment (cbrenner04) [inline server.js]: @a-ashbeck after changing this I was able to go to `/register` register and then login

File: app/config/routes.js (hunk 6 above)
Comment (cbrenner04) [reply app/config/routes.js]: I was able to register and login after my change but once I log in nothin is rendering except the boilerplate index.html

File: controllers/mainController.js (hunk 3 above)
Comment (cbrenner04) [reply controllers/mainController.js:1]: nevermind

File: server.js (hunk 4 above)
Comment (cbrenner04) [reply server.js:2]: nevermind

File: app/config/routes.js (hunk 6 above)
Comment (a-ashbeck) [reply app/config/routes.js]: the routes are just currently defined in the app. I figured we could abstract them out later and put them into this file

File: server.js (hunk 4 above)
Comment (a-ashbeck) [reply server.js:2]: hahahaha, I'd like to hear this story

**Comment by cbrenner04** [comment]
Ok. Couple of things. To answer my own questions... Some imported items are in `{}` and some are not due to the way they are exported from their original sources. From my understand the default class does not need `{}`. Also I didn't update the es5 stuff because of errors I was getting and ESLint isn't complaining.

**Review by cbrenner04 — APPROVED** [review]

**Pull request merged**